scripts/validate_docstrings.py

```diff
@@ -585,6 +586,20 @@ def validate_one(func_name):
 
     errs = []
     wrns = []
+
+    if len(doc.raw_doc) == 0:
```

Comment:
Can we just leverage the implicit truthiness here instead, so `if not doc.raw_doc` or maybe even just `if not doc`?